src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.stars__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars_star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 .stars_star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
```

Comment:
The image path should be '../images/star-active.svg'. Always ensure the path is relative to the location of the CSS file.